iatidatacube/import_data.py

```diff
@@ -270,8 +305,7 @@ def add_or_update_activity(activity_row):
 def get_reporting_org(reporting_orgs, row):
     ro_row = row['reporting_org#en']
     if ro_row not in reporting_orgs:
-        ro_row_ref = get_groups_or_none(re.match(r"(.*) \[(.*)\]",
-                ro_row), 1)
+        ro_row_ref = get_groups_or_none(re.match(r"(.*) \[(.*)\]", ro_row), 1)
```

Comment:
It would probably be useful if `iati-flattener` would add a column `reporting_org_ref` which would e.g. just contain `XM-DAC-46002` rather than `African Development Bank [XM-DAC-46002]` - I would imagine this line adds some unnecessary overhead?

Reply:
Yeah, could do. I'll add an issue.